docs/_templates/2019/lt/menu-mobile.html

```diff
@@ -0,0 +1,53 @@
+
+
+                <ul class="uk-nav uk-nav-primary uk-nav-center uk-margin-auto-vertical">
+                  <li><a href="{{ pathto('conf/vilnius/2019') }}">Home</a></li>
+                  <li><a href="{{ pathto('conf/vilnius/2019/news') }}">News</a></li>
+                  <li class="uk-parent">
+                      <a href="{{ pathto('conf/vilnius/2019/schedule') }}">Program</a>
+                      <ul class="uk-nav-sub">
+                          {% if flaglivestreaming %}
+                          <li><a href="{{ pathto('conf/vilnius/2019/livestream') }}">Live Stream</a></li>
+                          {% endif %}
+                          <li><a href="{{ pathto('conf/vilnius/2019/schedule') }}">Schedule</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/speakers') }}">Talks</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/lightning-talks') }}">Lightning Talks</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/unconference') }}">Unconference</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/writing-day') }}">Writing Day</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/job-fair') }}">Job Fair</a></li>
+                          <li><a href="{{ pathto('conf/vilnius/2019/badge-flair') }}">Badge Flair Contest</a></li>
+                          {% if flaghashike %}
+                          <li><a href="{{ pathto('conf/portland/2019/outing') }}">Hike</a></li>
```

Comment:
```suggestion
                          <li><a href="{{ pathto('conf/vilnius/2019/outing') }}">Hike</a></li>
```

Reply:
There won't be a hike AFAIK or another outing, can we remove this entirely?

Reply:
Sure, you probs want to delete lines 19-22 or wherever the condition ends, and `outing.rst` ;-)